src/components/TeamVacancies/TeamVacancies.tsx

```diff
@@ -15,11 +21,13 @@ const StyledVacancyList = styled.div`
 `;
 
 type TeamVacanciesProps = {
-    groupId: string;
+    group: Group & GroupSupervisor;
```

Comment:
here and in createModal

Reply:
here only groupId needed, I misses this

Reply:
no: here it also needed to pass in CreateVacancyModal